src/apiClient.js

```diff
@@ -2816,7 +2816,8 @@ class ApiClient {
     getUserViews(options = {}, userId) {
         const url = this.getUrl(`Users/${userId || this.getCurrentUserId()}/Views`, options);
 
-        return this.getJSON(url);
+        const json = this.getJSON(url);
+        return json;
```

Comment:
Why make this change?

Reply:
This was a temporary thing to get the value when testing. Needs to be removed.